README.md

```diff
@@ -1,125 +1,128 @@
-  <a title="Vercel" href="https://vercel.com">
-    <picture>
-      <source media="(prefers-color-scheme: dark)" srcset="https://img.shields.io/badge/powered%20by-Vercel%20%E2%96%B2-white">
-      <img src="https://img.shields.io/badge/powered%20by-Vercel%20%E2%96%B2-black" alt="Powered by Vercel">
-    </picture>
-  </a>
-  <br />
-  <img src="https://github.com/nodejs/nodejs.org/actions/workflows/build.yml/badge.svg" alt="Build and Analysis Checks" />
-  <a title="scorecard" href="https://securityscorecards.dev/viewer/?uri=github.com/nodejs/nodejs.org">
-    <img src="https://api.securityscorecards.dev/projects/github.com/nodejs/nodejs.org/badge" alt="nodejs.org scorecard badge" />
-  </a>
-  <br />
-  <br />
-</p>
-
-## What is this repo?
-
-[Nodejs.org](https://nodejs.org/) by the [OpenJS Foundation](https://openjsf.org/) is the official website for the Node.js® JavaScript runtime. This repo is the source code for the website. It is built using [Next.js](https://nextjs.org), a React Framework.
+  <p>
+    <a href="https://nodejs.org" target="_blank" rel="noopener noreferrer">Node.js</a> website built using Next.js, TypeScript, Tailwind CSS, and MDXv3.
+  </p>
+
+  <p>
+    <a href="LICENSE" title="MIT License">
+      <img src="https://img.shields.io/badge/license-MIT-blue" alt="MIT License" />
+    </a>
+    <a href="https://crowdin.com/project/nodejs-web" title="Localization" target="_blank" rel="noopener noreferrer">
+      <img src="https://badges.crowdin.net/nodejs-web/localized.svg" alt="Crowdin Badge" />
+    </a>
+    <a href="https://vercel.com" title="Powered by Vercel" target="_blank" rel="noopener noreferrer">
+      <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="https://img.shields.io/badge/powered%20by-Vercel%20%E2%96%B2-white" />
+        <img src="https://img.shields.io/badge/powered%20by-Vercel%20%E2%96%B2-black" alt="Powered by Vercel" />
+      </picture>
+    </a>
+  </p>
+
+  <p>
+    <img src="https://github.com/nodejs/nodejs.org/actions/workflows/build.yml/badge.svg" alt="Build and Analysis Checks" />
+    <a href="https://securityscorecards.dev/viewer/?uri=github.com/nodejs/nodejs.org" title="Security Scorecard" target="_blank" rel="noopener noreferrer">
+      <img src="https://api.securityscorecards.dev/projects/github.com/nodejs/nodejs.org/badge" alt="Security Scorecard" />
+    </a>
+  </p>
+</div>
+
+---
+
+## What is this repository?
+
+[Nodejs.org](https://nodejs.org/) is the official website for the Node.js® JavaScript runtime, maintained by the [OpenJS Foundation](https://openjsf.org/).  
+This repository contains the website source code, built with [Next.js](https://nextjs.org), TypeScript, Tailwind CSS, and MDXv3.
 
 ```bash
 pnpm install --frozen-lockfile
 pnpm dev
 
-# listening at localhost:3000
+# Available at http://localhost:3000
 ```
 
+---
+
 ## Contributing
 
-This project adopts the Node.js [Code of Conduct][].
+This project adheres to the Node.js [Code of Conduct][].
```

Comment:
I don't think `adheres to` is grammatically correct, we _adopt_ it as our code of conduct, which we then expect all contributors to _adhere to_.